suites/focal.toml

```diff
@@ -16,12 +16,12 @@ version = "1.67.2-1652812855"
 
 [[direct]]
 name = "discord"
-version = "0.0.90-1"
+version = "0.0.91"
 
     [[direct.urls]]
 #    url = "https://dl.discordapp.net/apps/linux/${version}/discord-${version}.deb"
     url = "https://www.dropbox.com/scl/fi/3at99qzffc9re726e7ddn/discord-0.0.90-custom.deb?rlkey=k4p6rz0xvkpo8hp36pz0symzo&st=9avqhsiu&dl=1"
```

Comment:
Since Discord pulled 0.0.90 and we had to implement [this workaround](https://github.com/pop-os/repo-proprietary/pull/129) to fall back to 0.0.89 with Debian packaging, this PR will have to remove this line and uncomment the line above it. Can you make that change?

Reply:
I did it with 716904b.